### PR TITLE
Portfolio: details popup window

### PR DESCRIPTION
**Pull request by Megagig**
### **The Following Features Were Implemented During This Milestone:**

- [ ] I use a JavaScript array to store all of the information for all projects
- [ ] All the projected were stored in an array of objects.
- [ ] I updated the main page so that the projects section is created dynamically using the information stored in that JavaScript object.

### CODE REVIEW

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @Megagig, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has a live demo link that leads to 404, i would implore you to remove it or you fix it.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Megagig** [comment]
Thanks so much, this is an oversight @Olamarx . Corrected with thanks

File: Styles.css [hunk 1]
```diff
@@ -1,59 +1,268 @@
...
+    border: 1px solid #ebecf0;
+    width: 90%;
+    margin: 5% auto 3% auto;
+    padding: 5px 20px;
+    justify-content: center;
+    top: 60px;
+    height: auto;
+    box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+    border-radius: 16px;
+  }
```
Comment (binhussen) [inline Styles.css]: - Great job for implementing the popup functionality 👍 however you didn’t style it as the given Figma design the popup image is not customized on the mobile version which leads to overflow, and the background is scrollable which leads to a bad User experience 👍 please fix it to stick the given design and have better UI
check [this video](https://www.loom.com/share/1ee32b62294040c38f880ef130dbb5b0) how your work looks

File: script.js [hunk 2]
```diff
@@ -1,24 +1,196 @@
...
+            </button>
+            <button class="submit-github">
+              <div class="live-version">
+                <a href="">See Source </a>
+              </div>
+              <div class="github-img">
+                <img src="images/Vector.png" alt="github icon" class="github-icon-popup" />
+              </div>
+            </button>
+          </div>
```
Comment (binhussen) [inline script.js]: - Kindly, the `see live`, `see source`, and `close` buttons have the default `a` style so update to stick to the Figma design 👍
### your design 👇

### Figma design 👇

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @rabayarabu , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 2 above)
Comment (Megagig) [reply script.js]: I have corrected this, thanks so much for pointing it out for me. @binhussen

File: Styles.css (hunk 1 above)
Comment (Megagig) [reply Styles.css]: Thanks for the loom video and clear explanation of what needed to be done. I have fixed this also. @binhussen

**Review by Strangeal — APPROVED** [review]
Hi @Megagig, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**